package/cloudshell/orch/training/logic/create_user_sandboxes.py

```diff
@@ -55,11 +56,11 @@ def _get_latest_sandbox_details(self, sandbox: Sandbox) -> ReservationDescriptio
         return sandbox_details.ReservationDescription
 
     def _send_emails(self):
-        if self._email_service.is_email_configured():
```

Comment:
I noticed that you removed this check. If email configurations are not passed the code will still try to send an email to users and then we will get an exception